public/app/features/manage-dashboards/state/actions.ts

```diff
@@ -182,6 +182,26 @@ const getDataSourceOptions = (input: { pluginId: string; pluginName: string }, i
   }
 };
 
+export async function moveFolders(folderUIDs: string[], toFolder: FolderInfo) {
+  const result = {
+    totalCount: folderUIDs.length,
+    successCount: 0,
+  };
+
+  for (const folderUID of folderUIDs) {
+    try {
+      const newFolderDTO = await moveFolder(folderUID, toFolder);
```

Comment:
could we use `Promise.allSettled` here instead of doing this sequentially? 🤔

Reply:
We _could_, but I purposefully made it sequential...

At least, it matches how the other bulk actions work

Reply:
hmmmm... is that just cause the other bulk actions were implemented before `Promise.allSettled` existed? 😅 if you're moving a large number of folders this could be sloooooow 

fine for an alpha anyway, we can optimise later 👍